handbook/tools/slack.md

```diff
@@ -73,6 +73,7 @@ of each category of channel by searching for a prefix!
   for Launch Pad leadership
 * [`#ask-design`](https://ubclaunchpad.slack.com/messages/C747550BD): Questions
```

Comment:
Unrelated, but do you mind tacking this to this PR?

```suggestion
* [`#ask-design`](https://ubclaunchpad.slack.com/messages/CK0H2GNQH): Questions
```

Reply:
sure